Sample.Hosted/buildwww/lib/react/components/HelloReactComponent.css

```
@@ -0,0 +1,107 @@
+    @apply text-center pb-8;
+}
+
+.drn-react-root .hello-react-badge {
+    @apply inline-block rounded-full px-4 py-1.5 text-xs font-semibold tracking-wider uppercase;
+    background: linear-gradient(135deg, #e0e7ff 0%, #c7d2fe 100%);
+    color: #3730a3;
+}
+
+.drn-react-root .hello-react-title {
+    @apply text-3xl font-bold tracking-tight mt-4 mb-3;
+    color: #1e293b;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)